frontend/public/components/utils/copy-to-clipboard.jsx

```diff
@@ -1,15 +1,65 @@
 import * as React from 'react';
 
 import { CopyToClipboard as CTC } from 'react-copy-to-clipboard';
+import { Tooltip } from 'react-lightweight-tooltip';
 
-export const CopyToClipboard = ({value, visibleValue = value}) => <React.Fragment>
-  <div className="co-copy-to-clipboard">
-    <pre className="co-pre-wrap co-copy-to-clipboard__text">{visibleValue}</pre>
-    <CTC text={value}>
-      <button className="btn btn-default co-copy-to-clipboard__btn" type="button">
-        <i className="fa fa-clipboard" aria-hidden="true"></i>
-        <span className="sr-only">Copy to Clipboard</span>
-      </button>
-    </CTC>
-  </div>
-</React.Fragment>;
+export class CopyToClipboard extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {copied: false};
+  }
+
+  render() {
+    const overrides = {
+      wrapper: {
+        position: 'absolute',
+        right: '0px',
+        top: '0px',
+        width: '42px',
+        height: '42px',
+      },
+      tooltip: {
+        left: '-90px',
+        bottom: '-6px',
+        maxWidth: '150px',
+        minWidth: '150px',
+        padding: '0',
+        textAlign: 'center',
+      },
+      content: {
+        display: 'block',
+        fontSize: '12px',
+        padding: '7px 12px',
+        whiteSpace: 'normal',
+      },
+      arrow: {
+        bottom: '8px',
+        left: '103%',
+        borderTop: '8px solid transparent',
+        borderBottom: '8px solid transparent',
+        borderLeft: '8px solid #000',
+      }
+    };
+
+    let text;
+    if (this.state.copied) {
+      text = <span className="co-nowrap" key="nowrap">Copied</span>;
+    } else {
+      text = <span className="co-nowrap" key="nowrap">Copy to Clipboard</span>;
+    }
+
+    return <React.Fragment>
+      <div className="co-copy-to-clipboard">
+        <pre className="co-pre-wrap co-copy-to-clipboard__text">{this.props.visibleValue}</pre>
+        <Tooltip content={text} styles={overrides}>
```

Comment:
Need to pass `text` as an array since it contains markup (e.g., `content={[text]}`).  Note the console error.

![screen shot 2018-06-28 at 9 34 50 am](https://user-images.githubusercontent.com/895728/42037513-906e0e46-7ab6-11e8-955f-bfd13020433f.PNG)